nomad/acl_endpoint.go

```diff
@@ -284,6 +276,19 @@ func (a *ACL) GetPolicy(args *structs.ACLPolicySpecificRequest, reply *structs.S
 	return a.srv.blockingRPC(&opts)
 }
 
+func (a *ACL) requestAuthToken(secretID string) (*structs.ACLToken, error) {
```

Comment:
This might be a little clearer?

```suggestion
func (a *ACL) requestACLToken(secretID string) (*structs.ACLToken, error) {
```